src/components/MovieList.js

```diff
@@ -0,0 +1,95 @@
+import styled from 'styled-components';
+
+import useStore from './../hooks/useStore';
+import Button from './Button';
+
+const Section = styled.section`
+	padding: 16px;
+`;
+
+const Article = styled.article`
+	max-width: 154px;
+	padding: 0;
+	animation: fadeIn 1s ${({delay}) => delay}s forwards;
+	border: 1px solid rgba(0, 0, 0, 0.2);
+	border-radius: 20px;
+	opacity: 0;
+	background-color: #fff;
+	box-shadow: 1px 1px 10px 5px rgba(0, 0, 0, 0.1);
+	color: #fff;
+
+	@keyframes fadeIn {
+		0% {
+			opacity: 0;
+		}
+		100% {
+			opacity: 1;
+		}
+	}
+`;
+
+const Picture = styled.img`
+	border-radius: 20px 20px 0 0;
+`;
+
+const Grid = styled.div`
+	display: flex;
+	flex-wrap: wrap;
+	gap: 16px;
```

Comment:
Your styled component is named "Grid" but you are using flexbox which is kinda confusing. I suggest choosing another name e.g. wrapper/container.

Reply:
renamed to Container